jsonrpc/filter_manager.go

```diff
@@ -82,8 +83,12 @@ type filterBase struct {
 
 // newFilterBase initializes filterBase with unique ID
 func newFilterBase(ws wsConn) filterBase {
+	uuidObj := uuid.New()
+	uuidBytes := uuidObj[:]
+	newID := "0x" + hex.EncodeToString(uuidBytes)
```

Comment:
You can use https://github.com/0xPolygon/polygon-edge/blob/develop/jsonrpc/types.go#L322 `encodeToHex`